src/wpantund/NCPInstanceBase.cpp

```diff
@@ -583,6 +587,13 @@ NCPInstanceBase::property_set_value(
 				reinitialize_ncp();
 			}
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_DaemonSetDefRouteForAutoAddedPrefix)) {
+			mSetDefaultRouteForAutoAddedPrefix = any_to_bool(value);
+			cb(0);
+			if (mTerminateOnFault && (get_ncp_state() == FAULT)) {
+				reinitialize_ncp();
```

Comment:
Good catch...Thanks... I have no idea how this was added here...will remove it...